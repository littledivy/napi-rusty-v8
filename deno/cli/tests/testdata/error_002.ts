import { throwsError } from "./subdir/mod1.ts";

function foo() {
  throwsError();
}

foo();
