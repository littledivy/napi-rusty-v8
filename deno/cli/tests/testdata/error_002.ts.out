[WILDCARD]error: Uncaught Error: exception from mod1
  throw Error("exception from mod1");
        ^
    at throwsError ([WILDCARD]/subdir/mod1.ts:16:9)
    at foo ([WILDCARD]/error_002.ts:4:3)
    at [WILDCARD]/error_002.ts:7:1
